packages/docs/src/locales/ru/ru.json

```
@@ -3848,5 +3855,15 @@
       "nuxtConfig": "Затем обновите файл nuxt-config.ts:",
       "moreAboutConfig": "Подробнее про конфигурирование"
     }
+  },
+  "spacer": {
+    "title": "Spacer component",
+    "summaryText": "`va-spacer` аналог для свойства `flex-grow`. Он позволяет добавить пустое пространсто между компонентами.",
+    "examples": {
+      "default": {
+        "title": "Базовое использование",
+        "text": "`va-spacer` добавляет div с классом `.spacer`."
```

Should be replaced in component
```suggestion
        "text": "`va-spacer` добавляет div с классом `.va-spacer`."
```